src/layout/json.rs

```diff
@@ -91,7 +81,7 @@ impl JsonLayout {
         record.key_values().visit(&mut visitor)?;
 
         let record_line = RecordLine {
-            timestamp: humantime::format_rfc3339_micros(SystemTime::now()),
+            timestamp: jiff::Zoned::now(),
```

Comment:
Seems the default format is not aligned:

```
2024-08-11T13:47:37.947497+00:00[UTC] ERROR simple_stdio: examples/simple_stdio.rs:32 Hello error!
2024-08-11T13:47:37.98353+00:00[UTC]  WARN simple_stdio: examples/simple_stdio.rs:33 Hello warn!
2024-08-11T13:47:37.983557+00:00[UTC]  INFO simple_stdio: examples/simple_stdio.rs:34 Hello info!
2024-08-11T13:47:37.983579+00:00[UTC] DEBUG simple_stdio: examples/simple_stdio.rs:35 Hello debug!
2024-08-11T13:47:37.983602+00:00[UTC] TRACE simple_stdio: examples/simple_stdio.rs:36 Hello trace!
```

Let's see ..

Reply:
This is probably a customization that would be okay, i.e., controlling the precision. If you want to file an issue for that, that would be great.

Otherwise I think here you could use `%Y-%m-%dT%H:%M:%S.%.6f%:z[%:V]`. (I wrote that from memory so double-check me there.) This is assuming you're okay with microsecond precision. You can use `%.9f` for nanosecond precision.

Reply:
@BurntSushi Thank you! `let time = Zoned::now().strftime("%Y-%m-%dT%H:%M:%S.%6f");` works.

> file an issue for that

I'd like to.

This display logic implements the std::fmt interface, how can we control it? Modifying a global state may be brittle.

Reply:
No... It wouldn't be adjusted via global state. It would be a new method on [`jiff::fmt::temporal::DateTimePrinter`](https://docs.rs/jiff/latest/jiff/fmt/temporal/struct.DateTimePrinter.html).

Reply:
Might also be able to integrate with: https://doc.rust-lang.org/std/fmt/struct.Formatter.html#method.precision

Reply:
Filed https://github.com/BurntSushi/jiff/issues/92.